$OpenBSD: patch-configure_ac,v 1.5 2013/11/21 17:00:32 ajacoutot Exp $

XXX: Short cut check for shm_open(3). Remove when we have librt.

--- configure.ac.orig	Sat Nov  9 15:44:36 2013
+++ configure.ac	Thu Nov 21 17:43:13 2013
@@ -929,6 +929,9 @@ AG_GST_CHECK_FEATURE(SHM, [POSIX shared memory source 
             AC_DEFINE(HAVE_OSX,[1],[Apple Mac OS X operating system detected])
             HAVE_SHM=yes
             ;;
+	*-*-openbsd*)
+	    HAVE_SHM=yes
+	    ;;
         *)
             AC_CHECK_LIB([rt], [shm_open],
                 AC_CHECK_DECL([MSG_NOSIGNAL], HAVE_SHM=yes, HAVE_SHM=no, [
